src/ssl.c

```diff
@@ -3099,6 +3099,7 @@ void* wolfSSL_GetDecryptVerifyCtx(WOLFSSL* ssl)
     return NULL;
 }
 
+#ifdef HAVE_ENCRYPT_THEN_MAC
```

Comment:
Also check for !defined(WOLFSSL_AEAD_ONLY).
Also, I think it should be around all the functions here from wolfSSL_CTX_SetEncryptMacCb() to wolfSSL_GetVerifyDecryptCtx()

Reply:
Thanks Sean! Will update it.